iothub/device/src/Common/InternalBufferManager.cs

```diff
@@ -330,7 +320,7 @@ public override void ReturnBuffer(byte[] buffer)
                 {
                     if (buffer.Length != bufferPool.BufferSize)
                     {
-                        throw Fx.Exception.Argument("buffer", CommonResources.BufferIsNotRightSizeForBufferManager);
+                        throw Fx.Exception.Argument(nameof(buffer), Common.Resources.BufferIsNotRightSizeForBufferManager);
```

Comment:
question - how did this change from `CommonResources` to `Common.Resources`?

Reply:
Common.Resources has the property that is implemented by CommonResources ... unnecessary change, but suggested by the IDE